providers/common/macs/gmac_prov.c

```diff
@@ -174,7 +174,8 @@ static int gmac_set_ctx_params(void *vmacctx, const OSSL_PARAM params[])
     OPENSSL_CTX *provctx = PROV_LIBRARY_CONTEXT_OF(macctx->provctx);
     const OSSL_PARAM *p;
 
-   if (!ossl_prov_cipher_load_from_params(&macctx->cipher, params, provctx))
+   if (!ossl_prov_cipher_load_from_params(&macctx->cipher, params, provctx)
+        || ctx == NULL)
```

Comment:
Is this really the right order?

Reply:
I doubt it matters, but I'll switch it around.